src/main/java/pl/edu/pja/nyan/service/WordService.java

```diff
@@ -90,4 +96,42 @@ public void delete(Long id) {
         log.debug("Request to delete Word : {}", id);
         wordRepository.deleteById(id);
     }
+
+    public void saveWordsIfNecessary(List<WordDTO> words, Tag lessonTag) {
```

Comment:
Nice, ten kawałek kodu jest super 👍

Reply:
Ewentualnie w przyszłości można by było go podzielić na kilka kawałków dla czytelności, bo trochę to zajmuje czasu żeby zrozumieć o co tu chodzi, ale mamy ważniejsze rzeczy do zrobienia